colossalai/inference/engine/engine.py

```diff
@@ -126,7 +126,7 @@ def __init__(
         # Init pg mesh
         pg_mesh = ProcessGroupMesh(pp_size, tp_size)
 
-        stage_manager = PipelineStageManager(pg_mesh, PP_AXIS, True)
+        stage_manager = PipelineStageManager(pg_mesh, PP_AXIS, True if pp_size * tp_size > 1 else False)
```

Comment:
`PipelineStageManager` is not related to tp. Why consider tp here?

Reply:
Cause hybridengine use pp and tp together with the same pipeline logic, which is also worked when pipeline size is 1.